app/src/main/java/ar/scacchipa/jsonplaceholderviewer/data/ITypicodeSourceData.kt

```diff
@@ -0,0 +1,27 @@
+package ar.scacchipa.jsonplaceholderviewer.data
+
+import com.google.gson.GsonBuilder
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface ITypicodeSourceData {
+    @GET("posts")
+    suspend fun getPosts(): Response<List<Post>>
+
+    @GET("posts/{id}/comments")
+    suspend fun getComment(@Path("id") id: String): Response<List<Comment>>
+}
+
+fun provideRetrofit(): Retrofit {
```

Comment:
I think this retrofit stuff deserves to be in a different file for its own.

Reply:
It was moved to another files.